modules/veteran/app/models/veteran/service/representative.rb

```diff
@@ -12,6 +12,10 @@ class Representative < ApplicationRecord
 
       validates_presence_of :poa
 
+      def self.find_match(user)
```

Comment:
I would rename this find by user

Reply:
@ATeal I used find match because of this rubocop error Rails/DynamicFindBy: Rails/DynamicFindBy: Use `find_by` instead of dynamic `find_by_user`. thinking its a column

Reply:
Hrm, If we were using this outside of claims, I would say refactor it onto the OpenIdUser Model, so you could treat it like a `has_one` relationship and call `user.representative` or something like that... `find_match` just isn't super descriptive, but I guess it's either that or ignore the rubocop error. Your choice!

Reply:
@ATeal I updated the method to for_user which I think covers descriptive side. Plus it is flexible enough that we do add the ssn and birthdate to the method to do a find best match like mvi does.